### PR TITLE
JQuery,DataTables,Bootstrap plugins added

### PR DESCRIPTION
**Pull request by mckiekun**
JQuery,DataTables,Bootstrap plugins added

### CODE REVIEW

File: application/views/templates/footer.php [hunk 1]
```diff
@@ -1,6 +1,22 @@
 
         </div>
     </div>
+
+ <!-- // ADDING BOOTSTRAP // -->
+
+ <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```
Comment (kento-sama) [inline application/views/templates/footer.php:7]: move this to header. Every CSS are linked in header file

File: jQueryDataTableBootstrap.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```
Comment (kento-sama) [inline jQueryDataTableBootstrap.txt]: para asa ni nga file? i-delete na lang guru ni.

Comment (mckiekun) [reply jQueryDataTableBootstrap.txt]: aw sige delete nko senpai kent. nagbutang2 rko ana hehe

**Review by melcrisopon — CHANGES_REQUESTED** [review]
Mckie deleted some files and request to change it

**Pull request closed without merging**